EIP712/Implementation/EIP712.cs

```diff
@@ -0,0 +1,203 @@
+﻿using EIP712.Attributes;
+using EIP712.Exceptions;
+using EIP712.Utilities;
+using Nethereum.ABI.Encoders;
+using Nethereum.Signer;
+using Nethereum.Util;
+using System;
+using System.Diagnostics;
+using System.Linq;
+using System.Numerics;
+using System.Reflection;
+using System.Text;
+
+namespace EIP712
+{
+    public static class EIP712
+    {
+
+        private static readonly byte[] _eip191Header = new byte[] { 0x19, 0x1 };
```

Comment:
I would make a `Constants.cs` file where I put all the constants (if the whole library is going to have like 5+ constants).